src/main/java/de/tum/in/www1/artemis/domain/TextExercise.java

```diff
@@ -35,17 +34,12 @@ public void setExampleSolution(String exampleSolution) {
         this.exampleSolution = exampleSolution;
     }
 
-    @JsonIgnore
-    public boolean isFeedbackSuggestionsEnabled() {
-        return getAssessmentType() == AssessmentType.SEMI_AUTOMATIC;
-    }
-
     /**
      * Disable feedback suggestions for this exercise by setting the assessment type to MANUAL.
      * Only changes the assessment type if feedback suggestions are currently enabled.
      */
     public void disableFeedbackSuggestions() {
```

Comment:
Why does this only seem to exist for text exercises and not for programming exercises?

Reply:
I removed the method